docker/server/Dockerfile.ubuntu

```diff
@@ -9,13 +9,14 @@ RUN sed -i "s|http://archive.ubuntu.com|${apt_archive}|g" /etc/apt/sources.list
     && groupadd -r clickhouse --gid=101 \
     && useradd -r -g clickhouse --uid=101 --home-dir=/var/lib/clickhouse --shell=/bin/bash clickhouse \
     && apt-get update \
+    && apt-get upgrade -yq \
```

Comment:
AFAIK, it's generally a bad idea to do so in the Dockerfile. Images are managed by the maintainers, and the result becomes unpredictable and non-reproducible because of that